WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -89,3 +114,14 @@ extension BlogDashboardViewController {
         static let interGroupSpacing: CGFloat = 8
     }
 }
+
+class DashboardCardCell: UICollectionViewCell, Reusable {
+    func configure(_ viewController: UIViewController, blog: Blog) {
+        let postsViewController = PostsCardViewController(blog: blog)
+        viewController.addChild(postsViewController)
+        contentView.addSubview(postsViewController.view)
+        postsViewController.view.translatesAutoresizingMaskIntoConstraints = false
+        contentView.pinSubviewToAllEdges(postsViewController.view)
+        postsViewController.didMove(toParent: viewController)
+    }
```

Comment:
What do you think of creating a separate file for this cell? Also, I wonder if we should call it `PostsCardCell` instead.

Reply:
Changed in c74616168c0b8d1bb5478a668baec550060863ad

`PostsCardCell` already exists, so I named it `DashboardPostsCardCell`. It's just a placeholder class, as this will evolve. But I want to be super clear on what belongs to the dashboard.